### PR TITLE
[ranking tile] add state code for USA place name

### PR DESCRIPTION
**Pull request by chejennifer**
add state code when states differ across the list of places:

stateCode

don't add state code if all places are from the same state:

stateCode2

### CODE REVIEW

**Review by beets — APPROVED** [review]
cool, thanks

File: static/js/utils/place_utils.ts [hunk 1]
```diff
@@ -196,6 +197,26 @@ export function getPlaceNames(
     });
 }
 
+/**
+ * Given a list of place dcids, returns a promise with a map of dcids to place
+ * display names (adds state code to the end of the name if available)
```
Comment (juliawu) [inline static/js/utils/place_utils.ts:202]: Please clarify the difference between `getPlaceNames()` and `getPlaceDisplayNames()`

**Review by juliawu — APPROVED** [review]
Nice!

File: static/js/utils/place_utils.ts (hunk 1 above)
Comment (chejennifer) [reply static/js/utils/place_utils.ts:202]: done

**Pull request merged**